bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -56,12 +57,21 @@ spec:
               exit 0
           }
           trap quit SIGTERM
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --system-id=random
+          setenforce Permissive
+          SYSTEMD_IGNORE_CHROOT=yes systemctl start openvswitch
           ovs-appctl vlog/set "file:${OVS_LOG_LEVEL}"
           /usr/share/openvswitch/scripts/ovs-ctl --protocol=udp --dport=6081 enable-protocol
 
           tail -F --pid=$(cat /var/run/openvswitch/ovs-vswitchd.pid) /var/log/openvswitch/ovs-vswitchd.log &
           tail -F --pid=$(cat /var/run/openvswitch/ovsdb-server.pid) /var/log/openvswitch/ovsdb-server.log &
+          SYSTEMD_IGNORE_CHROOT=yes systemctl is-active --quiet openvswitch
+          status=$?
+          if [[ "$status" == 0 ]]; then
+              echo "OK"
+          else
```

Comment:
@mccv1r0 could you grab the output of `systemctl status` when it fails?

Reply:
reissue `systemctl status` ?